services/actions-handler/handler/controller_tasks.go

```diff
@@ -70,7 +70,7 @@ func (m *Messenger) handleTask(ctx context.Context, messageQueue *mq.MessageQueu
 				updateProject.StandbyProductionEnvironment = &advTask.ProductionEnvironment
 			}
 			// update the project in the api
-			updatedProject, err := lagoon.UpdateProject(ctx, int(project.ID), updateProject, l)
+			updatedProject, err := lagoon.UpdateProject(ctx, uint(int(project.ID)), updateProject, l)
```

Comment:
`project.ID` is a `uint`, so you can remove both the `uint()` and `int()` types from this

```suggestion
			updatedProject, err := lagoon.UpdateProject(ctx, project.ID, updateProject, l)
```